### DISCUSSION TITLE
Chat

### DISCUSSION THREAD

**Discussion by Autofirejm** [vercte/forumoji#10] [General]
discuss pulling, merging, etc. for the repository
(please don't close)

**Comment by kccuber-scratch** [comment]
sorry, i moved it to a discussion. please don't use issues for chatting.

**Reply by Autofirejm** [reply]
ok i'm kinda new github anyway :P

**Reply by vercte** [reply]
I'm pretty new, but I've gotten the hang of the web client. Now to use it in a command line...

**Reply by Autofirejm** [reply]
i almost forgot that you can use github in a command line 😅

**Comment by vercte** [comment]
I think we should just do it on the specific merges/pulling/etc. There's comments there.
